### DISCUSSION TITLE
Issue with terminal font

### DISCUSSION THREAD

**Discussion by Allstreamer** [community/community#17062] [Codespaces]
This weird font rendering bug persists in the terminal even after refeshing or starting a new session:
![Screenshot (548)_LI](https://user-images.githubusercontent.com/48365544/169670688-29df99c0-c422-4d53-8dcc-71c988d65b3c.jpg)

Os: Windows 10
Browser: Brave: 1.38.119 Chromium: 101.0.4951.67 (64-bit)

**Comment by Allstreamer** [comment]
There is also this weird noise when selecting lines:
![Screenshot (550)](https://user-images.githubusercontent.com/48365544/169670850-d8aa69c7-1c61-4bd8-9110-50fdd5a82d56.png)

This is exclusive to the terminal, as far as i can see the other fonts are normal

**Comment by DenverCoder1** [comment]
See answer at https://github.com/github-community/community/discussions/16871#discussioncomment-2785108
